doc/decision-tree-widget/tree.yml

```diff
@@ -59,6 +59,11 @@ children:
         - label: based on custom logic
           children:
           - label: skipWhile
+      - label: from the end of the Observable
+        children:
+        - label: based on a given amount
+          children:
+          - label: skipLast
```

Comment:
Because there is only one child under "from the end of the Observable", I suggest you combine them into one. So:
```
- label: from the end of the Observable
  children:
  - label: skipLast
```